ui/src/app/views/workspace/component.ts

```diff
@@ -200,14 +207,16 @@ export class WorkspaceComponent implements OnInit, OnDestroy {
   }
 
   workspaceClusterBillingProjectId(): string {
-    if (this.workspace.namespace === this.freeTierBillingProject) {
+    if (this.useBillingProjectBuffer === undefined) {
```

Comment:
Took me a few minutes trying to figure out why we should return `null` in this case. Do you think it's worth adding a comment or adding a mini function to capture the context?

```
isServerConfigLoaded() { return this.useBillingProjectBuffer !== undefined } 
```

Reply:
Added a comment, this is the downside of getting flags from the server, though hopefully our React version of this will be a bit easier to use synchronously. In hindsight, we should have just provided an instance of the serverConfig for injection rather than using this service.